modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -106,6 +106,58 @@ definition {
 		}
 	}
 
+	@description = "LPS-180070 - Verify creating a Commerce Product entry triggers an action to add a user"
```

Comment:
@description = "[LPS-173537](https://issues.liferay.com/browse/LPS-173537) - Verify creating a custom object entry triggers an action to add an Account entry"